snap/snapcraft.yaml

```diff
@@ -58,25 +65,34 @@ parts:
       fi
 
       /usr/bin/dotnet publish src/Dotnet.Installer.Console --output "${SNAPCRAFT_PART_INSTALL}" \
-        --configuration Release -r "${RUNTIME_RID}" -p:DebugSymbols=false -p:DebugType=none
+        --configuration Debug -r "${RUNTIME_RID}" -p:DebugSymbols=true -p:DebugType=full
       chmod 555 "${SNAPCRAFT_PART_INSTALL}/Dotnet.Installer.Console"
-      
-      craftctl default
     override-stage: |
       craftctl default
-      # jq files
-      patchelf --set-interpreter /snap/dotnet/current/lib64/ld-linux-x86-64.so.2 usr/bin/jq
-      patchelf --force-rpath --set-rpath /snap/dotnet/current/usr/lib/x86_64-linux-gnu usr/bin/jq
-      # .NET Installer files
-      patchelf --set-interpreter /snap/dotnet/current/lib64/ld-linux-x86-64.so.2 Dotnet.Installer.Console
-      patchelf --force-rpath --set-rpath \$ORIGIN/netcoredeps:/snap/dotnet/current/usr/lib/x86_64-linux-gnu Dotnet.Installer.Console
-    prime:
-      - usr/bin/jq
-      - usr/lib/*/libjq.so*
-      - usr/lib/*/libonig.so*
-      - Dotnet.Installer.Console
-      - Configuration/*
-      - Scripts/*
```

Comment:
question: Is it intended that the `prime` section is removed?

Reply:
Yes, the `stage` step will already contain all of the necessary files to prime, so no need to specify this in here.

I originally had a `craftctl default` line in the `override-build` section, which was copying everything from `build` to `stage`, requiring a `prime:` section later on. This is no longer the case.